spec/rubocop/cop/sketchup_suggestions/command_title_spec.rb

```diff
@@ -0,0 +1,59 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+
+describe RuboCop::Cop::SketchupSuggestions::CommandTitle do
+
+  subject(:cop) { described_class.new }
+
+  bad_capitalization = [
+    'text',
+    'text text',
+    'Text text',
+    'text Text',
+    'Text.',
+    'Text Text.',
+    'Text text.',
+    'text 2-Point',
+    'text 2-point'
+  ]
+
+  bad_capitalization.each do |keyword|
+    it "registers an offense when using UI::Command.new(\"#{keyword}\")" do
+      expect_offense(<<~RUBY, keyword: keyword)
+        UI::Command.new("%{keyword}")
+        ^^^^^^^^^^^^^^^^^^{keyword}^^ Use title case in command titles. [...]
+      RUBY
+    end
+
+    # TODO: Add for setter methods.
+    # Can we test by how a local variable was defined? Or only by its name?
```

Comment:
Can the node matcher be used to match a local variable based on its previous definition?

```x = UI::Command.new("Bla bla") # Define a node matcher for this definition
x.menu_text = "Bla bla" # Match for any X, using "nested" node matcher```

Or is our best bet to hardcode typical variable names?

Reply:
It is possible to track variable names, though you have to do that manually. Because of that I've never done anything more than check the local scope. I think I've done something like this in the cops for `register_extension`.